src/app/pages/ArticlePage/experimentTopStories/helpers.test.tsx

```diff
@@ -116,74 +122,24 @@ describe('AMP top stories experiment', () => {
-            "topStoriesSecondaryColumnClick": "http://foobar.com?idclient=123-456-789&s=$IF($EQUALS($MATCH(\${ampGeo}, gbOrUnknown, 0), gbOrUnknown), ${destinationId}, ${gnlId})&s2=${producerId}&p=SOURCE_URL&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&atc=PUB-[article]-[top-stories-promo]-[VARIANT(topStoriesExperiment)]-[]-[SOURCE_URL]-[secondaryColumn]-[]-[]&mv_test=Google Discover&mv_experiment_id=topStoriesExperiment&mv_creation=VARIANT(topStoriesExperiment)&type=AT",
-            "topStoriesSecondaryColumnView": "http://foobar.com?idclient=123-456-789&s=$IF($EQUALS($MATCH(\${ampGeo}, gbOrUnknown, 0), gbOrUnknown), ${destinationId}, ${gnlId})&s2=${producerId}&p=SOURCE_URL&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&ati=PUB-[article]-[top-stories-section]-[VARIANT(topStoriesExperiment)]-[]-[SOURCE_URL]-[secondaryColumn]-[]-[]&mv_test=Google Discover&mv_experiment_id=topStoriesExperiment&mv_creation=VARIANT(topStoriesExperiment)&type=AT",
-          },
-          "triggers": {
-            "articleBodyPromoClick": {
-              "on": "click",
-              "request": "topStoriesArticleBodyClick",
-              "selector": "div[data-experiment-position='articleBody'] a[aria-labelledby*='top-stories-promo']",
-            },
-            "articleBodyView": {
-              "on": "visible",
-              "request": "topStoriesArticleBodyView",
-              "visibilitySpec": {
-                "continuousTimeMin": 200,
-                "selector": "div[data-experiment-position='articleBody'] > section[aria-labelledby='top-stories-heading']",
-                "totalTimeMin": 500,
-                "visiblePercentageMin": 20,
-              },
-            },
-            "secondaryColumnPromoClick": {
-              "on": "click",
-              "request": "topStoriesSecondaryColumnClick",
-              "selector": "div[data-experiment-position='secondaryColumn'] a[aria-labelledby*='top-stories-promo']",
-            },
-            "secondaryColumnView": {
-              "on": "visible",
-              "request": "topStoriesSecondaryColumnView",
-              "visibilitySpec": {
-                "continuousTimeMin": 200,
-                "selector": "div[data-experiment-position='secondaryColumn'] > section[aria-labelledby='top-stories-heading']",
-                "totalTimeMin": 500,
-                "visiblePercentageMin": 20,
-              },
-            },
-          },
-        }
-        `);
+        expect(analyticsConfig).toMatchSnapshot();
```

Comment:
I've used `toMatchSnapshot()` instead of `toMatchInlineSnapshot()` as it was getting a bit too long to manage/read easily. Let me know if this is not preferred or if there's a better way round it!